package.json

```diff
@@ -63,8 +63,8 @@
   "dependencies": {
     "axios": "^0.17.1",
     "element-ui": "^2.0.11",
-    "imperavi-kube": "^6.5.2",
     "openseadragon": "^2.3.1",
+    "rematrix": "^0.2.2",
```

Comment:
Just read up on this package, interesting! Looks to be helpful for the purposes of the combination panel.